weDevsOfficial/dokan
# update: `withdraw_limit` default changed to `0` from `-1`

### All Submissions:





### Changes proposed in this Pull Request:
```php
dokan_get_option( 'withdraw_limit', 'dokan_withdraw', 0 ),
```


* Related https://github.com/weDevsOfficial/dokan-pro/pull/2412

### How to test the changes in this Pull Request:

1. keep the default withdrawal limit blank,
2. Make a withdrawal request.
3. Check if any error occurred.



### Changelog entry

no change log needed


### PR Self Review Checklist:

* Code is not following code style guidelines
* Bad naming: make sure you would understand your code if you read it a few months from now.
* KISS: Keep it simple, Sweetie (not stupid!).
* DRY: Don't Repeat Yourself.
* Code that is not readable: too many nested 'if's are a bad sign.
* Performance issues
* Complicated constructions that need refactoring or comments: code should almost always be self-explanatory.
* Grammar errors.


### FOR PR REVIEWER ONLY:

> As a reviewer, your feedback should be focused on the idea, not the person. Seek to understand, be respectful, and focus on constructive dialog.

> As a contributor, your responsibility is to learn from suggestions and iterate your pull request should it be needed based on feedback. Seek to collaborate and produce the best possible contribution to the greater whole.
